pkg/rpcclient/unwrap/unwrap.go

```diff
@@ -236,6 +293,28 @@ func ArrayOfUint160(r *result.Invoke, err error) ([]util.Uint160, error) {
 	return res, nil
 }
 
+// ArrayOfUint256 checks the result for correct state (HALT) and then extracts a
+// slice of util.Uint256 from the returned stack item.
+func ArrayOfUint256(r *result.Invoke, err error) ([]util.Uint256, error) {
```

Comment:
Maybe then we'd also like to add `ArrayOfSignatures`? We can reuse `ArrayOfBytes` and add length check.

Reply:
It's `[][]byte` in the binding because we don't have any special `crypto/*` type for signatures, therefore it makes little sense adding anything else.